src/upgrade/renderer/UpgradeApp.vue

```diff
@@ -3,55 +3,92 @@
   - SPDX-License-Identifier: AGPL-3.0-or-later
 -->
 
-<script setup>
-import IconUpdate from 'vue-material-design-icons/Update.vue'
+<script setup lang="ts">
+import IconCloudDownload from 'vue-material-design-icons/CloudDownload.vue'
 import IconWeb from 'vue-material-design-icons/Web.vue'
 import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
-import { translate as t } from '@nextcloud/l10n'
+import { t } from '@nextcloud/l10n'
 import { generateUrl } from '@nextcloud/router'
 
 const packageInfo = window.TALK_DESKTOP.packageInfo
 
-const browserLink = generateUrl('apps/spreed')
+const browserLink = generateUrl('/apps/spreed')
 </script>
 
 <template>
 	<div class="upgrade">
-		<h2>{{ t('talk_desktop', 'Upgrade required') }}</h2>
-		<p>{{ t('talk_desktop', 'The client version is too old and no longer supported by this server. Update is required.') }}</p>
-		<NcButton type="primary"
-			wide
-			:href="packageInfo.repository"
-			target="_blank">
-			<template #icon>
-				<IconUpdate />
-			</template>
-			{{ t('talk_desktop', 'Update Talk Desktop ↗') }}
-		</NcButton>
-		<NcButton type="secondary"
-			wide
-			:href="browserLink">
-			<template #icon>
-				<IconWeb />
-			</template>
-			{{ t('talk_desktop', 'Continue in Web-Browser ↗') }}
-		</NcButton>
+		<div class="upgrade__content">
+			<h2 class="upgrade__heading">
+				{{ t('talk_desktop', 'Upgrade required') }}
+			</h2>
+			<p class="upgrade__text">
+				{{ t('talk_desktop', 'The client version is too old and no longer supported by this server. Update is required.') }}
+			</p>
+		</div>
+		<div class="upgrade__actions">
+			<NcButton :href="packageInfo.repository"
+				target="_blank"
+				type="primary"
+				wide>
+				<template #icon>
+					<IconCloudDownload :size="20" />
+				</template>
+				{{ t('talk_desktop', 'Update Talk Desktop ↗') }}
+			</NcButton>
+			<NcButton :href="browserLink"
+				type="secondary"
+				target="_blank"
+				wide>
+				<template #icon>
+					<IconWeb :size="20" />
+				</template>
+				{{ t('talk_desktop', 'Continue in Web-Browser ↗') }}
```

Comment:
```suggestion
				{{ t('talk_desktop', 'Continue in Web-Browser') }} ↗
```

Saving translators some pain? Same above?

But now I see this was part before so it would be a new string now.